demo

```diff
@@ -35,6 +36,13 @@ while [ -n "$1" ]; do
     shift
 done
 
+# Check if the demo-artifact has been downloaded,
+# or if there exists a newer one in the s3-storage.
+DEMO_ARTIFACT_NAME="core-image-full-cmdline-qemux86-64.mender"
+# wget -q --timestamping file://home/olepor/yocto/qemu-setup/build/tmp/deploy/images/qemux86-64/core-image-full-cmdline-qemux86-64.mender
```

Comment:
From what I've read it seems that `wget --timestamp` will do what is required, as long as the remote is up and running serving the demo-artifact.